include/lsst/jointcal/PhotomFit.h

```diff
@@ -17,113 +17,109 @@ namespace jointcal {
 
 class Associations;
 
-/*! Some comments.
-
-
-*/
-
-
 //! Class that handles the photometric least squares problem.
-
 class PhotomFit {
   private :
 
-  Associations &_assoc;
-  std::string _WhatToFit;
+  Associations &_associations;
+  std::string _whatToFit;
   bool _fittingModel, _fittingFluxes;
   unsigned _nParModel, _nParFluxes, _nParTot;
   PhotomModel * _photomModel;
   double _fluxError;
-  int _LastNTrip; // last triplet count, used to speed up allocation
-
-
+  int _lastNTrip; // last triplet count, used to speed up allocation
 
  public :
 
   //! this is the only constructor
-  PhotomFit (Associations &A, PhotomModel *M, double PosError);
+  PhotomFit (Associations &associations, PhotomModel *model, double fluxError);
 
   //! Does a 1 step minimization, assuming a linear model.
   /*! It calls AssignIndices, LSDerivatives, solves the linear system
     and calls OffsetParams. No line search. Relies on sparse linear
     algebra. */
-  bool Minimize(const std::string &WhatToFit);
+  bool minimize(const std::string &whatToFit);
 
   //! Derivatives of the Chi2
-  void LSDerivatives(TripletList &TList, Eigen::VectorXd &Rhs) const;
+  void LSDerivatives(TripletList &tripletList, Eigen::VectorXd &rhs) const;
 
-  //! Compute the derivatives for this CcdImage. The last argument allows to to process a sub-list (used for outlier removal)
-void LSDerivatives(const CcdImage &Ccd,
-		   TripletList &TList, Eigen::VectorXd &Rhs,
-		   const MeasuredStarList *M=nullptr) const;
+  //! Compute the derivatives for this CcdImage. The last argument allows to to
+  //! process a sub-list (used for outlier removal)
```

Comment:
I'm going to leave this one (grammar and all) alone until I understand it better (understanding that bit is outside scope for this ticket, methinks).